SonarQube.TeamBuild.Integration/SonarQube.TeamBuild.Integration.csproj

```diff
@@ -1,269 +1,57 @@
-    </Reference>
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.ServiceModel" />
-    <Reference Include="System.Threading.Tasks.Dataflow, Version=4.5.24.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.Tpl.Dataflow.4.5.24\lib\portable-net45+win8+wpa81\System.Threading.Tasks.Dataflow.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Web.Http, Version=5.2.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.AspNet.WebApi.Core.5.2.2\lib\net45\System.Web.Http.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
+    <None Remove="License.txt" />
   </ItemGroup>
+
   <ItemGroup>
     <Compile Include="..\AssemblyInfo.Shared.cs">
       <Link>Properties\AssemblyInfo.Shared.cs</Link>
     </Compile>
-    <Compile Include="BuildEnvironment.cs" />
-    <Compile Include="BuildSummaryLogger.cs" />
-    <Compile Include="BuildVNextCoverageReportProcessor.cs" />
-    <Compile Include="CoverageReportConverter.cs" />
-    <Compile Include="CoverageReportDownloader.cs" />
-    <Compile Include="CoverageReportProcessorBase.cs" />
-    <Compile Include="Interfaces\ITeamBuildSettings.cs" />
-    <Compile Include="Interfaces\IVisualStudioSetupConfigurationFactory.cs" />
-    <Compile Include="TfsLegacyCoverageReportProcessor.cs" />
-    <Compile Include="CoverageReportUrlProvider.cs" />
-    <Compile Include="Interfaces\ICoverageReportProcessor.cs" />
-    <Compile Include="Interfaces\ICoverageUrlProvider.cs" />
-    <Compile Include="Interfaces\ICoverageReportConverter.cs" />
-    <Compile Include="Interfaces\ICoverageReportDownloader.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="TeamBuildAnalysisSettings.cs" />
-    <Compile Include="TeamBuildSettings.cs" />
-    <Compile Include="TrxFileReader.cs" />
-    <Compile Include="VisualStudioSetupConfigurationFactory.cs" />
   </ItemGroup>
+
   <ItemGroup>
-    <ProjectReference Include="..\SonarQube.Common\SonarQube.Common.csproj">
-      <Project>{e312fdbb-bc13-4559-8f21-76ca5b88ebc5}</Project>
-      <Name>SonarQube.Common</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+    <Content Include="License.txt" />
```

Comment:
Same here - maybe keep the two _License.txt_ references together.

I'm not sure it's actually necessary to remove the _None_ reference - I don't think it will have any effect on the output (unless MSBuild tries to copy it to the _bin_ directory twice and fails on the second copy).